src/main/java/com/musala/atmosphere/client/util/ServerAnnotationProperties.java

```diff
@@ -2,58 +2,57 @@
 
 import org.apache.log4j.Logger;
 
-import com.musala.atmosphere.client.exceptions.MissingServerAnnotationException;
+import com.musala.atmosphere.client.exceptions.MissingServerConnectionProperiesException;
 
 /**
  * Container holding the properties of the first {@link Server} annotation found up in the stack trace.
- * 
+ *
  * @author yordan.petrov
- * 
+ *
  */
 public class ServerAnnotationProperties extends ServerConnectionProperties {
     private static final Logger LOGGER = Logger.getLogger(ServerAnnotationProperties.class.getCanonicalName());
 
-    private static final String LOGGER_ERROR_MESSAGE = "The invoking class is missing a @Server annotation.";
+    private static final String LOGGER_ERROR_MESSAGE = "The invoking class is missing a @Server annotation "
+            + "or missing config.properties in the test project working directory.";
```

Comment:
Why do you need to use "+" ? Also you can remove the second use of the word missing : "The invoking class is missing a @Server annotation or config.properties in the test project working directory.".